packages/itwin/tree-widget/api/tree-widget-react.exports.csv

```diff
@@ -7,18 +7,15 @@ beta;HierarchyVisibilityHandler
 beta;IModelContentTreeComponent:
 public;ModelsTreeComponent:
 beta;ModelsTreeVisibilityHandlerOverrides
-beta;ReportUsageCallback
 public;SelectableTree(props: SelectableTreeProps): JSX.Element | null
-beta;Tree({ getSchemaContext, hierarchyLevelSizeLimit, imodelAccess: providedIModelAccess, ...props }: TreeProps_3): JSX.Element
+beta;TelemetryContextProvider({ children, onPerformanceMeasured, onFeatureUsed, componentIdentifier, }: PropsWithChildren
+beta;Tree({ getSchemaContext, hierarchyLevelSizeLimit, selectionStorage, imodelAccess: providedIModelAccess, ...props }: TreeProps_3): JSX.Element
 public;TreeDefinition
-beta;TreeRenderer({ rootNodes, expandNode, onNodeClick, onNodeKeyDown, onNodeDoubleClick, isNodeSelected, onFilterClick, getIcon, getLabel, getSublabel, getHierarchyLevelDetails, checkboxProps, ...props }: TreeRendererProps_2): JSX.Element
+beta;TreeRenderer({ rootNodes, expandNode, onNodeClick, onNodeKeyDown, onNodeDoubleClick, isNodeSelected, onFilterClick, getIcon, getLabel, getSublabel, getHierarchyLevelDetails, checkboxProps, reloadTree, ...props }: TreeRendererProps_2): JSX.Element
 beta;TreeRendererProps = Required
-beta;TreeUsageTrackedFeatures = "hierarchy-level-filtering" | "hierarchy-level-size-limit-hit"
 public;TreeWidget
 public;TreeWidgetComponent(props: SelectableTreeProps): JSX.Element
-beta;useFeatureReporting
 beta;VisibilityStatus
-beta;VisibilityTree({ visibilityHandlerFactory, onPerformanceMeasured, treeRenderer, ...props }: VisibilityTreeProps_2): JSX.Element
-beta;VisibilityTreeRenderer({ getCheckboxState, onCheckboxClicked: onClick, ...props }: TreeRendererProps_3 & Pick
-beta;VisibilityTreeRendererProps = TreeRendererProps & Pick
-beta;VisibilityTreeUsageTrackedFeatures = TreeUsageTrackedFeatures | "visibility-change"
\ No newline at end of file
+beta;VisibilityTree({ visibilityHandlerFactory, treeRenderer, ...props }: VisibilityTreeProps_2): JSX.Element
+beta;VisibilityTreeRenderer({ getCheckboxState, onCheckboxClicked: onClick, ...props }: TreeRendererProps_3 & TreeCheckboxProps): JSX.Element
+beta;VisibilityTreeRendererProps = TreeRendererProps & Pick
```

Comment:
Do we really want to export `VisibilityTreeRendererProps` and `TreeRendererProps`?